cmd/neofs-adm/internal/modules/morph/generate.go

```diff
@@ -139,38 +141,55 @@ func refillGas(cmd *cobra.Command, gasFlag string, createWallet bool) error {
 	if err != nil {
 		return err
 	}
-	if storageWalletPath == "" {
-		return fmt.Errorf("missing wallet path (use '--%s <out.json>')", storageWalletFlag)
+	// wallet address is not part of the config
+	walletAddress, err := cmd.Flags().GetString(walletAddressFlag)
```

Comment:
error could be ignored here, it is non-nil only if the flag has not been defined